### ISSUE TITLE
The kube-apiserver bundled with kubebuilder should print out a valid version string

### ISSUE THREAD

**Issue by roberthbailey** [kubernetes-sigs/kubebuilder#518]
I was trying to troubleshoot an issue with the version of the kube-apiserver bundled with kubebuilder and wasn't able to see what version is being used. 

```
$ cd kubebuilder_1.0.5_darwin_amd64/bin/
$ ./kube-apiserver --version
Kubernetes v0.0.0-master+$Format:%h$
```

This issue was present when I looked at 1.0.3 and 1.0.5, but may be present in other releases.

**Comment by droot** [comment]
We are fixing this in next release. our current process builds all the binaries from the release-tag. Instead of building the binaries ourselves, we are going to just packages the binaries available on dl.k8s.io. 

/cc @mengqiy who is working on it.

**Comment by droot** [comment]
fixes #518
